index.html

```diff
@@ -81,6 +81,235 @@ <h4 class="connect font-style">LET'S CONNECT</h4>
+                    <h2 class="about-me-heading font-style">About <br> Myself</h2>
+                    <p class="description font-style">
+                        I’m a software developer! I can help you build a product , feature or
+                        website Look through some of my work and experience! If you like what
+                        you see and have a project you need coded, don’t hestiate to contact
+                        me.
+                    </p>
+                </section>
+
+                <section>
+
+                    <h4 class="connect font-style">LET'S CONNECT</h4>
+                    <ul class="socials">
+                        <li>
+                            <a class="social_medias twitter" href="https://twitter.com/KNurgi">
+                                <img src="images/Vector (1).svg" alt="twitter logo">
+                            </a>
+                        </li>
+
+                        <li>
+                            <a class="social_medias" href="https://www.linkedin.com/in/kenbon-teshome/">
+                                <svg xmlns="http://www.w3.org/2000/svg" width="23" height="23" fill="currentColor"
+                                    class="bi bi-linkedin icon" viewBox="0 0 16 16">
+                                    <path
+                                        d="M0 1.146C0 .513.526 0 1.175 0h13.65C15.474 0 16 .513 16 1.146v13.708c0 .633-.526 1.146-1.175 1.146H1.175C.526 16 0 15.487 0 14.854V1.146zm4.943 12.248V6.169H2.542v7.225h2.401zm-1.2-8.212c.837 0 1.358-.554 1.358-1.248-.015-.709-.52-1.248-1.342-1.248-.822 0-1.359.54-1.359 1.248 0 .694.521 1.248 1.327 1.248h.016zm4.908 8.212V9.359c0-.216.016-.432.08-.586.173-.431.568-.878 1.232-.878.869 0 1.216.662 1.216 1.634v3.865h2.401V9.25c0-2.22-1.184-3.252-2.764-3.252-1.274 0-1.845.7-2.165 1.193v.025h-.016a5.54 5.54 0 0 1 .016-.025V6.169h-2.4c.03.678 0 7.225 0 7.225h2.4z" />
+                                </svg>
+                            </a>
+                        </li>
+                        <li>
+                            <a class="social_medias" href="">
+                                <img src="images/mediumLogo.png" alt="Medium icon" class="medium" />
+                            </a>
+                        </li>
+                        <li>
+                            <a class="social_medias" href="https://github.com/KenbonTN">
+                                <img src="images/githublogo.png" alt="Github icon" />
+                            </a>
+                        </li>
+                        <li>
+                            <a href="#"><img src="images/Vector (2).svg" alt="two fingers"></a>
+                        </li>
+                    </ul>
+                    <button type="button" class="resume-button button">Get my resume</button>
+                </section>
+            </section>
+            <div class="languages_for_web">
+                <section class="languages">
```

Comment:
- [ ] Kindly consider the technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li`